src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java

```diff
@@ -26,25 +26,10 @@ protected AdsFandomObject loadPage(String pageName, String pageType) {
     }
 
     AdsFandomObject pageObject = new AdsFandomObject(driver, pageUrl);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "pageObject". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)